middleware/csrf/index.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const express = require('express');
+
+const csrf = require('csurf');
+
+const csrfProtection = csrf({
+    cookie: false,
+    sessionKey: 'cicaSession'
+});
+
+const app = express();
+app.use(csrfProtection);
+
+// error handler
+app.use((err, req, res, next) => {
```

Comment:
Is this a duplicate of the error handler in the new error-handler middleware? If so are they both required?